package.json

```diff
@@ -13,13 +13,23 @@
     "build": "tsc --p ./tsconfig-build.json && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
```

Comment:
Didn't notice this script in the last PR, do we need it? I think this is only needed for npm publishing

Reply:
happy to remove it 🙂

Reply:
also the "dev" script I don't think we need anymore as we have deleted the App. Will remove that too